File: backend/internal/core/domain/project.go
package domain

import "time"

type Project struct {
	ID            string     `json:"id"`
	Name          string     `json:"name"`
	Owner         int        `json:"owner"`
	SDKAuthSecret string     `json:"sdk_auth_secret"`
	CreatedAt     *time.Time `json:"created_at"`
	UpdatedAt     *time.Time `json:"updated_at"`
}

File: dashboard/src/api/dto/project.ts
export interface CreateProjectRequest {
    name: string;
}

export interface CreateProjectResponse {
        created_at: string;
        id: string;
        name: string;
        owner: number;
        updated_at: string;
}


export interface Project {
    id: string;
    name: string;
    owner: number;
    updated_at: string;
    created_at: string;
}
